Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/BaseAutonomous.java

```diff
@@ -1,4 +1,240 @@
+        motorFR.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+        motorBL.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+        motorBR.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+
+        // reset the timeout time and start motion.
+        runtime.reset();
+        motorFL.setPower(0.5);
+        motorFR.setPower(0.5);
+        motorBL.setPower(0.5);
+        motorBR.setPower(0.5);
+        // keep looping while we are still active, and there is time left, and both motors are running.
+        // Note: We use (isBusy() && isBusy()) in the loop test, which means that when EITHER motor hits
+        // its target position, the motion will stop.  This is "safer" in the event that the robot will
+        // always end the motion as soon as possible.
+        // However, if you require that BOTH motors have finished their moves before the robot continues
+        // onto the next step, use (isBusy() || isBusy()) in the loop test.
+        while (opModeIsActive() &&
+                (runtime.seconds() < 30) &&
+                (motorFL.isBusy() && motorFR.isBusy() && motorBL.isBusy() && motorBR.isBusy())) {
+        }
+
+        // Stop all motion;
+        stopDriving();
+
+        // Turn off RUN_TO_POSITION
+        motorFL.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorFR.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorBL.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        motorBR.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+    }
+    
+    // using imu
+    public void imuPivot(double referenceAngle, double targetAngle, double maxSpeed, double kAngle, double timeout) {
+        runtime.reset();
+        // counter-clockwise is positive
+        double pivot;
+        double currentRobotAngle;
+        double angleError;
+
+        targetAngle = referenceAngle + targetAngle;
+        targetAngle = adjustAngles(targetAngle);
+        do {
+            currentRobotAngle = imu.getAngularOrientation().firstAngle;
+            angleError = currentRobotAngle - targetAngle;
+            angleError = adjustAngles(angleError);
+            pivot = angleError * kAngle;
+
+            if (pivot >= 0.0) {
+                pivot = Range.clip(pivot, 0.15, maxSpeed);
+            } else {
+                pivot = Range.clip(pivot, -maxSpeed, -0.15);
+            }
+
+            speedFL = pivot;
+            speedFR = pivot;
+            speedBL = pivot;
+            speedBR = pivot;
+
+            motorFL.setPower(speedFL);
+            motorFR.setPower(speedFR);
+            motorBL.setPower(speedBL);
+            motorBR.setPower(speedBR);
+            idle();
+        }
+        while ((opModeIsActive() && (Math.abs(angleError) > 3.0)) && (runtime.seconds() < timeout));
+        stopDriving();
+    }
+
+    private void stopDriving() {
+        motorFL.setPower(0.0);
+        motorFR.setPower(0.0);
+        motorBL.setPower(0.0);
+        motorBR.setPower(0.0);
+
+    }
+
+    // normalizing the angle to be between -180 to 180
```

Comment:
javadoc comment here too

Reply:
Since you made a javadoc comment, you need to remove the regular comment and put that information inside the javadoc comment. You can't have both.